edge-modules/api-proxy-module/templates/nginx_default_config.conf

```diff
@@ -26,9 +26,7 @@ http {
 
         chunked_transfer_encoding on;
 
-        #if_tag ${NGINX_DEFAULT_TLS}	
         ssl_protocols ${NGINX_DEFAULT_TLS};
-        #endif_tag ${NGINX_DEFAULT_TLS}
```

Comment:
Assuming this should be removed now that NGINX_DEFAULT_TLS is a default value

Reply:
Yes this is correct. It will always be true.